src/Saunter/AsyncApiOptions.cs

```diff
@@ -12,54 +11,101 @@ namespace Saunter
-        
+        public Func<MemberInfo, string> PropertyNameSelector { get; set; } =
+            prop => JsonNamingPolicy.CamelCase.ConvertName(prop.Name);
+
         /// <summary>
-        /// A list of filters that will be applied to the generated AsyncAPI document.
+        ///     A list of filters that will be applied to the generated AsyncAPI document.
         /// </summary>
         public IList<IDocumentFilter> DocumentFilters { get; } = new List<IDocumentFilter>();
 
         /// <summary>
-        /// A list of filters that will be applies to any generated channels.
+        ///     A list of filters that will be applies to any generated channels.
         /// </summary>
         public IList<IChannelItemFilter> ChannelItemFilters { get; } = new List<IChannelItemFilter>();
-        
+
         /// <summary>
-        /// A list of filters that will be applied to any generated Publish operations.
+        ///     A list of filters that will be applied to any generated Publish operations.
         /// </summary>
         public IList<OperationFilter> OperationFilters { get; } = new List<OperationFilter>();
 
         /// <summary>
-        /// Options related to the Saunter middleware
+        ///     Options related to the Saunter middleware
         /// </summary>
         public AsyncApiMiddlewareOptions Middleware { get; set; } = new AsyncApiMiddlewareOptions();
     }
 
     public class AsyncApiMiddlewareOptions
     {
         public const string AsyncApiMiddlewareDefaultRoute = "/asyncapi/asyncapi.json";
+        private const string AsyncApiUiMiddlewareBasePath = "/asyncapi/ui/";
+        private const string AsyncApiUiMiddlewareDefaultRoute = "/asyncapi/ui/index.html";
+        private const string AsyncApiPlaygroundBaseUri = "https://playground.asyncapi.io/";
 
         /// <summary>
-        /// The route which the AsyncApi document will be hosted
+        ///     The route which the AsyncApi document will be hosted
         /// </summary>
         public string Route { get; set; } = AsyncApiMiddlewareDefaultRoute;
+
+        /// <summary>
+        ///     The route which the AsyncApi UI will be hosted
+        /// </summary>
+        public string UiRoute { get; set; } = AsyncApiUiMiddlewareDefaultRoute;
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public string UiBaseRoute { get; set; } = AsyncApiUiMiddlewareBasePath;
+
+        /// <summary>
+        ///     
+        /// </summary>
+        public string PlaygroundBaseAddress { get; set; } = AsyncApiPlaygroundBaseUri;
+
+        /// <summary>
+        ///     Key/Value pairs where keys will be replaced by the corresponding value in the html
+        ///     returned from playground
+        /// </summary>
+        public Dictionary<string, string> HtmlProxyRewrites { get; set; } = new Dictionary<string, string>
+        {
+            {
+                "text-xs uppercase text-grey mt-10 mb-4 font-thin",
```

Comment:
Can you explain why these changes are made?

Reply:
The styles being served for some of the content appears to be styled to show up in playground where there is an additional style sheet that sets some darker backgrounds. This results in the html generated by the generator serving some light-grey font on a light background.
I made use of this functionality to rewrite some of the styling to fix that, but ultimately I could see this being more formalized to enable some theme-ing of the AsyncApiUI.